scripts/validate_container.sh

```diff
@@ -59,6 +59,19 @@ installed_backends=`\
     do basename $file | cut -d "." -f 1; \
     done`
 
+declare -A backend_options=()
+for file in $(ls $CUDA_QUANTUM_PATH/targets/*.yml); do
```

Comment:
This will break if `$CUDA_QUANTUM_PATH` has a space in it. I don't expect that to be the case, but you never know 🙂 
```suggestion
for file in "$CUDA_QUANTUM_PATH"/targets/*.yml; do
```